### ISSUE TITLE
First release

### ISSUE THREAD

**Issue by sualko** [littleredbutton/bigbluebutton-api-php#28]
I think it's time for our first release and therefore we have to decide a view points:

- Do we agree that this is a feature release? (2.0.8 - 2.1.0)
- Should we decouple this repo from upstream? (It's not mandatory, but during merges it always suggests to merge into upstream)
- Who wants to be the release manager? (I can do it if you like)

Did I forget anything?

**Comment by sualko** [comment]
After #29 is merged, I would register the package on packagist and setup the push hook, but this is independent from the task of an release manager.

**Comment by sualko** [comment]
Here you go: https://packagist.org/packages/littleredbutton/bigbluebutton-api-php#dev-master

**Comment by SamuelWei** [comment]
- I would wait until #26 is done and do a feature release.
> Should we decouple this repo from upstream? (It's not mandatory, but during merges it always suggests to merge into upstream)

- @sualko I don't get the point, sorry. Could you explain it again, thanks.

- I'm fine with you being the release manager.

**Comment by joisarjignesh** [comment]
@sualko  i m okay with you as release manager.

**Comment by sualko** [comment]
Ok, thanks. I will tag the first release after #26 is merged.

@SamuelWei I added a webhook for packagist, because I don't like to give applications wide-ranging permissions.

**Comment by Yurujai** [comment]
> I think it's time for our first release and therefore we have to decide a view points:
> 
> * Do we agree that this is a feature release? (2.0.8 - 2.1.0)

- 2.1.0 sounds good :) but I would wait for #26 and #30.

> * Should we decouple this repo from upstream? (It's not mandatory, but during merges it always suggests to merge into upstream)

- The only advantage I see if we leave the reference is that it gives us a bit more of visibility. 

> * Who wants to be the release manager? (I can do it if you like)
> 
> Did I forget anything?

- I'm fine with you being the release manager.

**Comment by sualko** [comment]
> I don't get the point, sorry. Could you explain it again, thanks.

At the top you see "forked from", because the idea is that we merge our changes back to upstream. Therefore Github always recommend the upstream repository if you open a pr, the number of forks it the number of upstream forks and there all statistics are related to the upstream. It's not urgent and maybe bbb will change there mind later.

**Comment by SamuelWei** [comment]
I understand, yes we should decouple

**Comment by SamuelWei** [comment]
What about my comment here https://github.com/littleredbutton/bigbluebutton-api-php/issues/15#issuecomment-633044687 ?
Shouldn't we also implement this before this release?

**Comment by sualko** [comment]
While writing the changelog, I was thinking that the deprecation of php < 7.2 is a breaking change. So we should bump to 3. Any thoughts?

> We should also add the guest parameter to the JoinMeetingParameters, otherwise the guest settings are ignored.

Good point. I create a pr asap. Ref: https://docs.bigbluebutton.org/dev/api.html#join

**Comment by SamuelWei** [comment]
I think we can support php 7.2 until 30 Nov 2020 (end of security support). Our current version is working with php 7.2. It should be up to the developer to decide wether it is a good idea to choose a version close to end-of-life or not.

**Comment by sualko** [comment]
> I think we can support php 7.2 until 30 Nov 2020

Are you referring to my comment above? It's about the next version, because usually if you have a breaking change you bump the major version number, see https://semver.org

**Comment by SamuelWei** [comment]
Thanks! Sorry, I didn't get it. I thought you want to move to php 7.3 as minimal requirement.
We should bump to 3.

**Comment by sualko** [comment]
Done :tada:
